src/scss/variables/_syntax.scss

```diff
@@ -0,0 +1,9 @@
+$drops--prefix: 'c' !default;
+$drops--syntax: (
```

Comment:
set the variable by default as null in order to make the global configuration running by default